.gitignore

```diff
@@ -1,6 +1,7 @@
 # Built application files
 *.apk
 *.ap_
+output-metadata.json
```

Comment:
What's this for?

Reply:
When you build an APK, it generates that json to go alongside it.